ERCS/erc-7743.md

```diff
@@ -0,0 +1,68 @@
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-07-13
+---
+
+## Abstract
+This ERC proposes a new standard for non-fungible tokens (NFTs) that supports multiple owners. The MO-NFT standard allows a single NFT to have multiple owners, reflecting the shared and distributable nature of digital assets. This model also incorporates a mechanism for value depreciation as the number of owners increases, maintaining the principle that less ownership translates to more value.
+
+## Motivation
+Traditional NFTs enforce a single-ownership model, which does not align with the inherent duplicability of digital assets. MO-NFTs allow for shared ownership, promoting wider distribution and collaboration while maintaining secure access control. This model supports the principle that some valued information is more valuable if fewer people know it, hence less ownership means higher value.
+
+## Specification
+
+### Token Creation and Ownership Model
+1. **Minting**: When a digital asset is uploaded, a unique MO-NFT is minted, and the uploader becomes the initial owner.
+2. **Ownership List**: The MO-NFT maintains a list of owners. Each transfer adds the new owner to the list while retaining the existing owners.
+3. **Transfer Mechanism**: Owners can transfer the token to new owners. The transfer does not remove the original owner from the list but adds the new recipient.
+
+### Transfer of Ownership
+1. **Additive Ownership**: Transferring ownership adds the new owner to the ownership list without removing the current owners.
+2. **Ownership Tracking**: The smart contract tracks the list of owners for each MO-NFT.
+
+### Decryption Rights and Access Control
+1. **Encrypted Asset**: The digital asset associated with the MO-NFT is encrypted.
+2. **Owner Access**: Only addresses listed as owners in the smart contract can access the decryption key.
+
+### Value Depreciation
+1. **Value Model**: As the number of owners increases, the value of the MO-NFT decreases to reflect the reduced exclusivity.
+2. **Depreciation Mechanism**: The value depreciation model is defined based on the asset type and distribution strategy. Less ownership equates to more value, following the principle that valued information or assets become less valuable as they become more widely known or accessible.
+
+### Interface Definitions
+Define the necessary interfaces for interacting with MO-NFTs. This includes minting, transferring, and accessing ownership data.
+
+### [ERC-721](erc-721.md) Compliance
+Ensure compatibility with the existing ERC-721 standard for NFTs to maintain interoperability with existing tools and platforms.
+
+## Rationale
+The rationale behind MO-NFTs is to align the ownership model of NFTs with the nature of digital assets that can be easily copied and shared. By allowing multiple owners and implementing a value depreciation mechanism, MO-NFTs provide a more flexible and realistic approach to digital asset ownership.
```

Comment:
Your rationale is justifying the proposal as a whole. This section is intended to explain decisions made within the proposal itself. My favourite analogy is:

> Motivation: We need to build a shed because...
> Rationale: We painted the shed red because...